flutter/lib/benchmark/benchmark.dart

```diff
@@ -332,7 +357,8 @@ class BenchmarkState extends ChangeNotifier {
               benchmark.benchmarkSetting.writeToBuffer();
             }
           }
-        } catch (_) {
+        } catch (e) {
+          print(e);
```

Comment:
Could you revert this change, please?
It conflicts with https://github.com/mlcommons/mobile_app_open/pull/191.
It would be easier to revert it here than to resolve merge conflicts later.

Reply:
Ok